multi-casskop/deploy/crds/multicluster_v1alpha1_cassandramulticluster_cr.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: db.orange.com/v1alpha1
+kind: MultiCasskop
+metadata:
+  name: MultiCasskop-demo
+spec:
+  # Add fields here
+  base:
+    apiVersion: "db.orange.com/v1alpha1"
+    kind: "CassandraCluster"
+    metadata:
+      name: cassandra-demo
+      namespace: cassandra-demo
+      labels:
+        cluster: casskop
+    spec:
+      resources:         
+        requests:
+          cpu: '0.3'
+          memory: 2Gi
+        limits:
```

Comment:
You can simplify the yaml like I did in the past
```
requests: &requests
  cpu: '0.3'
  memory: 2Gi
limits: *requests
```

Reply:
this file is generated